src/app/app.component.ts

```diff
@@ -382,6 +384,7 @@ export class CopayApp {
           });
       });
     }
+    this.analyticsProvider.logEvent('adsupport_test_event', {});
```

Comment:
I would add it into that conditional: `this.platformProvider.isCordova && this.appProvider.info.name === 'bitpay'`